i18n-embed-fl/Cargo.toml

```diff
@@ -32,3 +32,7 @@ doc-comment = { workspace = true }
 env_logger = { workspace = true }
 pretty_assertions = { workspace = true }
 rust-embed = { workspace = true }
+
+[features]
+# Enables more performant version of the crate. May download extra dependencies.
+performance = ["dep:dashmap"]
```

Comment:
I'd call this feature `dashmap` as it just enables/disables the optional dependency and doesn't apply any other "performance" tweaks.